pyproject.toml

```diff
@@ -34,7 +34,7 @@ dependencies = [
     "lxml",
     "owlready2",
     "psycopg",
-    "python-dotenv",
+    "pydantic_settings",
```

Comment:
Is pydantic_settings following semver? If so then let's pin to the current major.

Reply:
Done.